sync/class.jetpack-sync-client.php

```diff
@@ -134,17 +134,21 @@ private function init() {
 		add_action( 'set_user_role', array( $this, 'save_user_role_handler' ), 10, 3 );
 		add_action( 'remove_user_role', array( $this, 'save_user_role_handler' ), 10, 2 );
 
-
 		// user capabilities
 		add_action( 'added_user_meta', array( $this, 'save_user_cap_handler' ), 10, 4 );
 		add_action( 'updated_user_meta', array( $this, 'save_user_cap_handler' ), 10, 4 );
 		add_action( 'deleted_user_meta', array( $this, 'save_user_cap_handler' ), 10, 4 );
 
-		// themes
+		// updates
 		add_action( 'set_site_transient_update_plugins', $handler, 10, 1 );
 		add_action( 'set_site_transient_update_themes', $handler, 10, 1 );
 		add_action( 'set_site_transient_update_core', $handler, 10, 1 );
 
+		// plugins
+		add_action( 'upgrader_process_complete', $handler, 10, 2 );
+		add_filter( 'jetpack_sync_before_send_upgrader_process_complete', array( $this, 'expand_upgrader_process_complete' ) );
+		add_action( 'deleted_plugin', $handler, 10, 2 );
+
```

Comment:
'installed plugin' doesn't exist as a hook https://developer.wordpress.org/?s=installed_plugin 

activation and deactivation is stored when the site option active_plugins changes.